test/test_suite.py

```diff
@@ -1110,6 +1143,9 @@ def test_numeric_as_float(self):
             [15.7563],
         )
 
+    @pytest.mark.skipif(
+        bool(os.environ.get("SPANNER_EMULATOR_HOST")), reason="Skipped on emulator"
+    )
```

Comment:
These two float tests don't contain null values so they aren't affected by the lack of null value type inference like the other tests. Additionally, they seem to be passing on the other PR:  https://github.com/cloudspannerecosystem/python-spanner-sqlalchemy/pull/85/checks?check_run_id=2849752522

So I don't think they should be skipped